apps/nextjs/src/components/ContextProviders/ChatProvider.tsx

```diff
@@ -113,26 +113,12 @@ export function ChatProvider({ id, children }: Readonly<ChatProviderProps>) {
     data: chat,
     isLoading: isChatLoading,
     refetch: refetchChat,
-  } = trpc.chat.appSessions.getChat.useQuery(
-    { id },
-    {
-      refetchOnMount: true,
-      refetchOnWindowFocus: true,
-      staleTime: 0,
-    },
-  );
+  } = trpc.chat.appSessions.getChat.useQuery({ id });
   const {
     data: moderations,
     isLoading: isModerationsLoading,
     refetch: refetchModerations,
-  } = trpc.chat.appSessions.getModerations.useQuery(
-    { id },
-    {
-      refetchOnMount: true,
-      refetchOnWindowFocus: true,
-      staleTime: 0,
-    },
-  );
+  } = trpc.chat.appSessions.getModerations.useQuery({ id });
```

Comment:
The refetch on mount option I think was what fixed the issue with back navigation when returning to the chat after the download page.

Reply:
You can't actually see this error in local development. You'll need to deploy to Vercel to see it in action, so could you check it doesn't reintroduce the problem?

Reply:
Sorry, I thought I had replied to this!

I looked at the tanstack-query docs which said that these three properties were already the defaults, so I did remove them without checking that. I'll have a test now and will see if I can get to the bottom of it